python/Ganga/Runtime/bootstrap.py

```diff
@@ -1279,7 +1279,8 @@ def launch_IPython(local_ns, args, error_handler, ganga_prompt):
         ipshell.confirm_exit = config['confirm_exit']
 
         # Launch embedded shell
-        ipshell(local_ns=local_ns)
+        from Ganga import GPI
+        ipshell(local_ns=local_ns, module=Ganga.GPI)
```

Comment:
Certainly originally we had GPI in both locals and globals and I personally think it makes sense to have it in globals as that's what I would naively expect as a user (e.g. so you can create jobs in functions). Now what in particular should be in the local_ns and (further up this function), what's passed to running the `.ganga.py`, etc. and returned is very much up for debate. From a quick look at the code it wasn't clear what the user would expect so we thought a quick fix for now followed by ore discussion later :smile: